src/nidigital/system_tests/test_system_nidigital.py

```diff
@@ -464,3 +455,42 @@ def test_get_pattern_pin_names(multi_instrument_session):
     pattern_pin_names = multi_instrument_session.get_pattern_pin_names(start_label='new_pattern')
 
     assert pattern_pin_names == ['LO' + str(i) for i in range(8)] + ['HI' + str(i) for i in range(8)]
+
+
+# nidigital specific converter tests
+# We are specifically using a "private" module to get to the converters so we can test the
+# nidigital specific one
+def test_convert_site_string():
+    test_result = nidigital._converters.convert_site_string('1')
+    assert test_result == 'site1'
+    test_result = nidigital._converters.convert_site_string(1)
+    assert test_result == 'site1'
+    test_result = nidigital._converters.convert_site_string('site1')
+    assert test_result == 'site1'
+    test_result = nidigital._converters.convert_site_string('42')
+    assert test_result == 'site42'
+    test_result = nidigital._converters.convert_site_string(42)
+    assert test_result == 'site42'
+    test_result = nidigital._converters.convert_site_string('site42')
```

Comment:
aren't strings such as  "0-2" and "site0-site2" and "0:2" valid? We should test them.

Reply:
No. Those are not valid. This converter is for parameters that take a single site only. If those are valid, then it would be set as a repeated capability.